gini/gini-switch-sdk-ios
# Feature optional extractions complete vc

# Introduction

Some users might have their own "Extractions complete" screen, so it should be possible to **not** show ours.

I've added the `shouldShowAnalyzingCompleteScreen` variable to `GiniSwitchConfiguration` to does just that.

# How to test

Set `shouldShowAnalyzingCompleteScreen` to false and see if the "Extractions complete" screen shows.

# Merge Info

Automatic
